tests/Application/config/sylius/1.11/bundles.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    SyliusLabs\Polyfill\Symfony\Security\Bundle\SyliusLabsPolyfillSymfonySecurityBundle::class => ['all' => true],
```

Comment:
Why we have only the two bundles in Sylius 1.11?

Reply:
Changed - moved BabDev\PagerfantaBundle to specific sylius version bundles